cmd-info.c

```diff
@@ -506,11 +506,12 @@ static int read_taskinfo(void *arg)
 {
 	struct ftrace_file_handle *handle = arg;
 	struct uftrace_info *info = &handle->info;
-	char buf[4096];
 	int i, lines;
 	int ret = -1;
 
-	if (fgets(buf, sizeof(buf), handle->fp) == NULL)
+	char* buf = NULL;
+	size_t len = 0;
```

Comment:
A minor coding-style nit-picking.

Please put declarations together at the beginning of a block.  I prefer keeping a blank line between declarations and statements.  Probably it's an old-style, but I used to it and it helps me understanding code a bit.  Thanks.